munich-university-of-applied-sciences-department-of-applied-social-sciences.csl

```diff
@@ -0,0 +1,329 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="de-DE">
+  <info>
+    <title>Munich University of Applied Sciences - Department of Applied Social Sciences</title>
+    <title-short>MUAS</title-short>
+    <id>http://www.zotero.org/styles/munich-university-of-applied-sciences-department-of-applied-social-sciences</id>
+    <link href="http://www.zotero.org/styles/munich-university-of-applied-sciences-department-of-applied-social-sciences" rel="self"/>
+    <link href="http://www.zotero.org/styles/taylor-and-francis-harvard-x" rel="template"/>
+    <link href="https://w3-mediapool.hm.edu/mediapool/media/fk11/fk11_lokal/dokumente_73/FirstSpirit_1507823177843Standards_Wiss_Arbeiten_2017_Schoenberger.pdf" rel="documentation"/>
```

Comment:
I would link to "http://www.sw.hm.edu/publikationen/lehrmaterialien/schoenberger.de.html" here, which is a much simpler and likely more stable URL (without a year in it).